sections/errorhandling/centralizedhandling.md

```diff
@@ -44,7 +42,12 @@ function errorHandler() {
     await logger.logError(err);
     await sendMailToAdminIfCritical;
     await saveInOpsQueueIfCritical;
-    await determineIfOperationalError;
+
+    const isOperationalError = await determineIfOperationalError;
+    if (!isOperationalError) {
+      // it's a programmer error, crash to recover the application
+      process.exit(1);
```

Comment:
Is this the best way to crash the application? Will process managers such as Forever and PM2 catch it correctly? I haven't tested it yet.

Reply:
@Calinou

Process.exit(1) is the way to do it.

Bottom part should be:
```
await saveInOpsQueueIfCritical;
    const isOperationalError = await determineIfOperationalError(err);
    if (!isOperationalError) {
      // it's a programmer error, crash to recover the application
      process.exit(1);
    }
```

Your first call to determineIfOperationalError seems redundant, no?